tests/shipping/models/Delivery - Peru.model.js

```diff
@@ -0,0 +1,60 @@
+import { visitAndClearCookies, setup } from '../../../utils'
+import {
+  getRandomEmail,
+  fillEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import { SKUS, ACCOUNT_NAMES } from '../../../utils/constants'
+import { selectCountry, goToPayment } from '../../../utils/shipping-actions'
+import { payWithBoleto, completePurchase } from '../../../utils/payment-actions'
+
+export default function test(account) {
+  describe(`Delivery - Peru - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only delivery', () => {
+      const email = getRandomEmail()
+
+      setup({ skus: [SKUS.GLOBAL_PRODUCT], salesChannel: 2, account })
+
+      fillEmail(email)
+      fillProfile()
+
+      selectCountry('PER')
+
+      if (account === ACCOUNT_NAMES.GEOLOCATION) {
+        cy.waitAndGet('#ship-addressQuery', 3000).type('Av. Javier Prado Este')
+
+        cy.get('.pac-item')
+          .first()
+          .trigger('mouseover')
+
+        cy.get('.pac-item')
+          .first()
+          .click()
+
+        cy.contains('Avenida Javier Prado Este')
+        cy.get('#ship-number').type('2465')
+      } else {
```

Comment:
yeah, but this part is only relevant because of the input method on geolocation and non-geolocation stores, for the other test stores we have we could reuse this same test as is and (I think) it should work

Reply:
Fine